test/npcomp-run-mlir/conv_2d_nchw.mlir

```diff
@@ -0,0 +1,40 @@
+// RUN: npcomp-run-mlir %s \
+// RUN:   -invoke conv_2d_nchw \
+// RUN:   -arg-value="dense<[[[[1.2]]]]> : tensor<1x1x1x1xf32>" \
+// RUN:   -arg-value="dense<[[[[3.4]]]]> : tensor<1x1x1x1xf32>" \
+// RUN:   -shared-libs=%npcomp_runtime_shlib 2>&1 \
+// RUN:   | FileCheck %s
+
+// RUN: npcomp-run-mlir %s \
```

Comment:
can you test more shapes to exercise various corners of the shape calculation logic? just use dense<0> as the inputs. This is mainly to verify that we aren't reading out of bounds or allocating the wrong sizes. Outputs should be all zeros in every case.

Reply:
Working on it...